api/advisory/polygonize_geotiff.py

```diff
@@ -0,0 +1,38 @@
+import sys
```

Comment:
I'd like this function to move to api/app/auto_spatial_advisory/polygonize.py - I think splitting out into api/advisory was a mistake (my bad - premature separation of code?).

Can you please move it there? Side by side with the other `polygonize` function it's also very clear what re-factoring and renaming we're going to have to do.